### DISCUSSION TITLE
Restart of ical.net

### DISCUSSION THREAD

**Discussion by axunonb** [ical-org/ical.net#594] [Announcements]
@rianjs, the founder of the **ical.net** project, and @axunonb have successfully transitioned ical.net to a new home under the new GitHub organization **ical-org**. After a period of inactivity, **ical.net** is set to be revitalized.

With over 15 million downloads on nuget.org, including 7 million for the latest version 4.2.0 alone, **ical.net** boasts a broad user base. In recent years, some DevOps have developed their own forks of **ical.net** and released NuGet packages. We hope, they'll come back **ical.net** in this project..

We aim to continue these various efforts and further enhance **ical.net** under **ical-org**.

Moving the repository to **ical-org** will not result in any significant changes: the MIT license still applies, and the package on nuget.org remains unchanged.

Additionally, @rianjs will continue as a maintainer of ical.net, providing support as his time allows. We extend our heartfelt thanks to him for his many years of dedication in shaping ical.net into a wonderful project.

If you have suggestions or want to provide code, you're very welcome.

**Comment by rianjs** [comment]
🎉 🎉 🎉

**Comment by DaleyKD** [comment]
I just wanted to tell you guys thank you so very much for reviving this.

Right around the time of release of .NET 5, I had a project that needed to utilize RRULEs and found this library. However, at the time, it didn't support .NET 5 and I needed to use someone's fork.

A few weeks ago (years later), another project I'm on was investigating the use of designing recurring patterns and we specifically decided to try cron expressions since this project wasn't alive.

The other day, we found that cron wouldn't support our use case, and seeing that this was alive again was like "a sign." We're going back to RRULEs, baby!!

Again, I can't thank you enough.
